mtgjson4/compile_mtg.py

```diff
@@ -84,36 +84,54 @@ def build_output_file(sf_cards: List[Dict[str, Any]], set_code: str) -> Dict[str
 def uniquify_duplicates_in_set(cards: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
     """
     For cards with multiple printings in a set, we need to identify
-    them against each other. We will add (a), (b), ... to the end
+    them against each other.
+    For silver border sets, we will add (a), (b), ... to the end
     of the card name to do so.
+    For non-silver bordered sets, we will create a "variations"
+    field will be created that has UUID of repeat cards
     :param cards: Cards to check and update for repeats
     :return: updated cards list
     """
-    unique_list = []
-    duplicate_cards: Dict[str, int] = {}
-    for card in cards:
-        # Only if a card is duplicated in a set will it get the (a), (b) appended
-        total_same_name_cards = len(
-            [item for item in cards if item["name"] == card["name"]]
-        )
+    if cards[0].get("borderColor", None) == "silver":
```

Comment:
This assumes the input is given homogenous border color - is it conceivable that it wouldn't be?

Reply:
You are correct that there could be non-silver, but I haven't seen a set where card 1 is non-silver. If you can think of a better way to check `(if any card has borderColor == silver, do this)` lmk

Reply:
Reasonable enough. As long as you can guarantee that all cards passed in belong to the same set, that's passable.